src/panels/lovelace/components/hui-confirmation-editor.ts

```diff
@@ -0,0 +1,164 @@
+import "@polymer/paper-input/paper-input";
+import "@polymer/paper-item/paper-icon-item";
+import "@polymer/paper-item/paper-item-body";
+
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  internalProperty,
+  LitElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+} from "lit-element";
+import memoizeOne from "memoize-one";
+import { assert } from "superstruct";
+
+import { fireEvent } from "../../../common/dom/fire_event";
+import "../../../components/ha-help-tooltip";
+import "../../../components/ha-service-picker";
```

Comment:
Not used?

Reply:
I think I'll be removing this editor as a whole based on your feedback